recipes/wil/all/conandata.yml

```diff
@@ -0,0 +1,9 @@
+sources:
+  "1.0.230202.1":
+    url: "https://github.com/microsoft/wil/archive/refs/tags/v1.0.230202.1.tar.gz"
+    sha256: "7bf01e9d93fb93f0fe2614492fac4a423b3a97b435015db74f5ac4a0270ebc8a"
+patches:
+  "1.0.230202.1":
+    - patch_file: "patches/1.0.230202.1-0001-fix-type-conversion.patch"
+      patch_description: "Explicitly writing the type instead of using decltype"
```

Comment:
Is there an issue or patch for this? 

This does not seem be fixing anything related to conan so the type is confusing